doc/examples/animations/lib/main.dart

```diff
@@ -16,14 +17,24 @@ void main() async {
 }
 
 class MyGame extends BaseGame with TapDetector {
-  final animation = SpriteAnimation.sequenced(
-    'chopper.png',
-    4,
-    textureWidth: 48,
-    textureHeight: 48,
-    stepTime: 0.15,
-    loop: true,
-  );
+  Image chopper;
+  Image creature;
+  SpriteAnimation animation;
+
+  @override
+  Future<void> onLoad() async {
+    chopper = await Flame.images.load('chopper.png');
+    creature = await Flame.images.load('creature.png');
+
+    animation = SpriteAnimation.sequenced(
```

Comment:
similarly, we can have a SpriteAnimation.loadSequenced static method. for every constructor overload we had, we can have a static async helper method

Reply:
same thing here.